pkgs/data/themes/catppuccin-gtk/create-theme.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/scripts/create_theme.py b/scripts/create_theme.py
+index 074dc43..7e7a8eb 100644
+--- a/scripts/create_theme.py
++++ b/scripts/create_theme.py
+@@ -10,7 +10,7 @@ from scripts.utils import replacetext, zip_multiple_folders
+ from scripts.var import def_color_map, repo_dir, src_dir, theme_name, work_dir
+ 
+ 
+-def create_theme(types: List[str], accents: List[str], dest: str, link: bool = False, 
++def create_theme(types: List[str], accents: List[str], dest: str, link: bool = False,
```

Comment:
What does this line do?

Reply:
Nothing, my editor had cleaned up whitespace seemingly, I removed this from the patch.